guides/DevOps/Jenkins.md

```diff
@@ -0,0 +1,10 @@
+# What is Continuous Integration
+Is a set of pratices to allow a system to be compiled,built and tested, development in independent environment.
+
+# Overview
+Jenkins is a Continuous Integration web application used to test and create artifacts of a software project.
+
+# How does Jenkins work ?
+Jenkins enables to generate packages at every predetermined time interval, for every commit in your repository, or just when you wish, you define the best way to your business.
```

Comment:
Expand this article. This is just a very small text.

Reply:
hi @the-ethan-hunt I made the adjustments requested and includes some more concepts, thanks for the review

I used these references too: 

https://www.thoughtworks.com/pt/insights/blog/putting-continuous-delivery-practice-go
https://jenkins.io/doc/pipeline/tour/getting-started/